.github/workflows/cypress.yml

```diff
@@ -0,0 +1,15 @@
+name: End-to-end tests
+on:
+  schedule:
+    # Runs at 10h and 22h everyday
+    - cron: '0 10,22 * * *'
```

Comment:
I guess this one needs to be merged to take effect 🤔

Reply:
If so, please just merge it on your own.

Reply:
I merged it into main

Reply:
Seems like it doesn't work now, will take a look.

Reply:
@ntt261298 You can try to test in "on push" before running periodically.